src/components/common/section-templates/single-title-quintuple-description/SingleTitleQuintupleDescription.module.scss

```
@@ -213,17 +213,21 @@ $desc-input-height: 115px;
         color: c.$grey-700;
     }
 
-    :global(textarea) {
+    :global(.char-limit-textarea__field) {
         width: $desc-input-width;
         height: $desc-input-height;
         box-sizing: border-box;
 
         background-color: c.$white;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)